packages/grid/_modules_/grid/components/GridViewport.tsx

```diff
@@ -78,6 +78,7 @@ export const GridViewport: ViewportType = React.forwardRef<HTMLDivElement, {}>(
             cellTabIndex={cellTabIndex}
             isSelected={selectionState[id] !== undefined}
             editRowState={editRowsState[id]}
+            className={options.classes?.cell}
```

Comment:
In the core the `className` prop is often applied to the root element, but `GridRowCells` has no root. 

```suggestion
            cellClassName={options.classes?.cell}
```